code/game/gamemodes/blob/powers.dm

```diff
@@ -267,8 +301,107 @@
 	if(!surrounding_turfs.len)
 		return
 
-	for(var/mob/living/simple_animal/hostile/blobspore/BS in living_mob_list)
+	for(var/mob/living/simple_animal/hostile/blob/blobspore/BS in living_mob_list)
 		if(isturf(BS.loc) && get_dist(BS, T) <= 35)
 			BS.LoseTarget()
 			BS.Goto(pick(surrounding_turfs), BS.move_to_delay)
-	return
\ No newline at end of file
+	return
+
+
+/mob/camera/blob/verb/split_consciousness()
+	set category = "Blob"
+	set name = "Split consciousness (100) (One use)"
+	set desc = "Expend resources to attempt to produce another sentient overmind"
+
+	if(!blob_nodes || !blob_nodes.len)
+		src << "<span class='warning'>A node is required to birth your offspring...</span>"
+		return
+	var/obj/effect/blob/node/N = locate(/obj/effect/blob) in blob_nodes
+	if(!N)
+		src << "<span class='warning'>A node is required to birth your offspring...</span>"
+		return
+
+	if(!can_buy(100))
+		return
+
+	verbs -= /mob/camera/blob/verb/split_consciousness //we've used our split_consciousness
```

Comment:
MAKE SURE TO REMOVE THIS FROM THE OFFSPRING TOO

I tried enabling this on tg without removing from the offspring, it was horrifying.